tests/Phinx/Db/Adapter/MysqlAdapterTest.php

```diff
@@ -1572,6 +1572,29 @@ public function testDumpBulkinsert()
         $this->assertEquals(0, $res[0]['COUNT(*)']);
     }
 
+    public function testDumpTransaction()
+    {
+        $inputDefinition = new InputDefinition([new InputOption('dry-run')]);
+        $this->adapter->setInput(new ArrayInput(['--dry-run' => true], $inputDefinition));
+
+        $consoleOutput = new BufferedOutput();
+        $this->adapter->setOutput($consoleOutput);
+
+        $this->adapter->beginTransaction();
+        $table = new \Phinx\Db\Table('table1', [], $this->adapter);
+
+        $table->addColumn('column1', 'string')
+            ->addColumn('column2', 'integer')
+            ->addColumn('column3', 'string', ['default' => 'test'])
+            ->save();
+        $this->adapter->commitTransaction();
+        $this->adapter->rollbackTransaction();
+
+        $actualOutput = $consoleOutput->fetch();
+        $this->assertStringStartsWith("START TRANSACTION;\n", $actualOutput, 'Passing the --dry-run doesn\'t drump the transaction to the output');
+        $this->assertStringEndsWith("COMMIT;\nROLLBACK;\n", $actualOutput, 'Passing the --dry-run doesn\'t drump the transaction to the output');
```

Comment:
_dump_ here and few more tests

Reply:
I'm not sure what you mean by this? Do you want me to test over the entire output message, and then should the other dump tests be changed to do that as well?

And for more tests, what did you have in mind that you'd like to see tested? This test (plus the dump ones before it) should cover the simple bases of "multiple executes in one query" and then the various other adapter tests cover that my removing the semi-colons from the functions hasn't broken anything.

Reply:
You wrote "drump" there, a typo.

Reply:
Oh, totally misread the intention of that statement, sorry! Fixed the typo.